src/models/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../config/config.js')[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
```

Comment:
Let's set and use environment variables across platforms with `cross-env`.
The method used here could introduce bugs later in the code.

Reply:
> Let's set and use environment variables across platforms with `cross-env`.
> The method used here could introduce bugs later in the code.

@Segun-Ogundipe. You could expatiate more. But I dont think so. **config.use_env_variable** is a one-off implementation by sequelize used to access connection string in config/config.js file. We dont need it for anything during our development. But if you have further reservations please let me know.

Reply:
> > Let's set and use environment variables across platforms with `cross-env`.
> > The method used here could introduce bugs later in the code.
> 
> @Segun-Ogundipe. You could expatiate more. But I dont think so. **config.use_env_variable** is a one-off implementation by sequelize used to access connection string in config/config.js file. We dont need it for anything during our development. But if you have further reservations please let me know.

As the code progresses, we'll be setting lots of environment variables for different platforms (test, development, staging, production). I feel it's more prudent to get that settled right now as it'll also be best practice to handle setting variables for different platforms with the **cross-env** package. These are just suggestions.